### PR TITLE
fixing the klog message issue

### PR DESCRIPTION
**Pull request by rkamudhan**
**What type of PR is this?**
> Uncomment only one, leave it on its own line:
>
> /kind api-change
 /kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Comment by fedebongio** [comment]
Is there an Issue this PR is fixing?

**Comment by liggitt** [comment]
@dims is this something anyone who copied sample controller previously would have to update when upgrading to 1.13? Did that make it into release notes?

File: staging/src/k8s.io/sample-controller/main.go [hunk 1]
```diff
@@ -38,6 +38,7 @@ var (
 )
 
 func main() {
+	klog.InitFlags(nil)
```
Comment (justinsb) [inline staging/src/k8s.io/sample-controller/main.go]: Note that where there are any remnants of glog floating around, we'll need to do something like https://github.com/kubernetes-sigs/cluster-api/pull/610

@dims do you think we should put something like that into klog?

Comment (dims) [reply staging/src/k8s.io/sample-controller/main.go]: yep, opened an issue so i won't forget https://github.com/kubernetes/klog/issues/26

**Review by 0xmichalis — APPROVED** [review]

**Pull request closed without merging**